### DISCUSSION TITLE
Is it still going on here at all...

### DISCUSSION THREAD

**Discussion by Menne01** [easytarget/esp32-cam-webserver#258] [Ideas]
The wish for Pan&Tilt has already been expressed here, but apparently there is no interest in ideas and improvements. I consider this thread to be asleep...there will be nothing new to come. What a pity about this project....

**Comment by easytarget** [comment]
Yes, But I took a summer recess.

**Comment by easytarget** [comment]
Actually; I had some bigger priorities for the last few years, so nothing happened here from me at all. Sorry for that but I had to make my own choices.

Anyway, I will wrap this up; I am not doing any further development on this project. it is old code and somewhat obsolete. I am currently working on retiring it; I will try to link to a more actively updated fork when I do.
